### DISCUSSION TITLE
Can Saved Filters include Coordinate Groups?

### DISCUSSION THREAD

**Discussion by stuntguy3000** [mattieserver/netbox-topology-views#351] [Q&A]
Would love to include Coordinate Groups as a Saved Filter item - this would streamline working through many topologies with different devices.

**Comment by mattieserver** [comment]
The `Coordinate Groups` are indeed not saved with the Saved Filters, i will see if we can add this.

**Reply by stuntguy3000** [reply]
I hope so - this would be amazing for usability.
